go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	github.com/stretchr/testify v1.7.0
 	github.com/tinkerbell/rufio v0.0.0-20220606134123-599b7401b5cc
 	github.com/tinkerbell/tink v0.6.1-0.20220509141453-30fe9e015575
+	github.com/whilp/git-urls v1.0.0
```

Comment:
Thanks for the contribution. Are there any alternatives to this lib - it doesn't seem to be maintained? We could also consider copying the functions we need into our repository.

Reply:
How about a regex to validate the ssh url?